features/testbot/via.feature

```diff
@@ -148,6 +148,24 @@ Feature: Via points
             | a,d,c     | abc,bd,bd,bd,abc,abc |
             | c,d,a     | abc,bd,bd,bd,abc,abc |
 
+    # See issue #2349
```

Comment:
I feel I added something like this in https://github.com/Project-OSRM/osrm-backend/pull/1899/files did this get removed by accident again?